pyproject.toml

```diff
@@ -22,7 +22,6 @@ python = "^3.9"
 pyroute2 = "*"
 pyyaml = "*"
 lxml = "*"
-libvirt-python = "*"
```

Comment:
poetry does also have optional dependencies so that could be something to look into but more for the bigger pr for optional dependencies that samuel wants to work on.

The point for this MR was to just start the effort/see how it could look like for making some library an optional dependency.

I removed the `pyproject.toml` change from this for now, which means that libvirt still remains a dependency from poetrys point of view, but not from lnst point of view.